edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/DummyModuleManager.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Agent.Core;
+    using Microsoft.Azure.Devices.Edge.Agent.Edgelet;
+    using Microsoft.Azure.Devices.Edge.Agent.Edgelet.Models;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class DummyModuleManager : IModuleManager
+    {
+        public Task CreateModuleAsync(ModuleSpec moduleSpec) => throw new NotImplementedException();
+
+        public Task StartModuleAsync(string name) => throw new NotImplementedException();
+
+        public Task StopModuleAsync(string name) => throw new NotImplementedException();
+
+        public Task DeleteModuleAsync(string name) => throw new NotImplementedException();
+
+        public Task RestartModuleAsync(string name) => throw new NotImplementedException();
+
+        public Task UpdateModuleAsync(ModuleSpec moduleSpec) => throw new NotImplementedException();
+
+        public Task UpdateAndStartModuleAsync(ModuleSpec moduleSpec) => throw new NotImplementedException();
+
+        public Task<SystemInfo> GetSystemInfoAsync() => Task.FromResult(new SystemInfo("kubernetes", "amd64", "v1"));
```

Comment:
nit: Based on this implementation, this class should be likely named as "KubernetesDummyModuleManager"

Reply:
It's already in the Kubernetes.IntegrationTest namespace...